cmd/convox/start.go

```diff
@@ -64,31 +73,33 @@ func cmdStart(c *cli.Context) error {
 	}
 
 	if pcc, err := m.PortConflicts(); err != nil || len(pcc) > 0 {
-		if err != nil {
-			stdcli.ExitError(err)
+		if err == nil {
+			err = fmt.Errorf("ports in use: %v", pcc)
```

Comment:
if the error is nil then and there ARE port conflicts we create a new validation error and exit with that
if the error is not nil then something went wrong parsing the manifest and we error out with that instead

QOSEventSend exits out if there was an error